python/spear/log.py

```diff
@@ -5,9 +5,9 @@
 import inspect
 import os
 
-def log(message):
+def log(*message):
     current_frame = inspect.currentframe()
-    print(_log_get_prefix(current_frame) + message)
+    print(_log_get_prefix(current_frame) + ', '.join([str(x) for x in message]))
```

Comment:
Don't add a space between arguments by default. That's not how `SP_LOG` works, and adding spaces is a less general interface that could be inconvenient when attempting to concatenate strings. Yes, it is sometimes desirable to add spaces, but it is always easy to do that at the call site.

Reply:
Updated it